doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc

```diff
@@ -0,0 +1,142 @@
+[id='jws-jdbc-config_{context}']
+= Configuring JDBC Web Server data sources on {JWS}
+
+Java Database Connectivity (JDBC) is an API specification used to connect programs written in Java to the data in popular databases.  A data source is an object that enables a JDBC client, such as an application server, to establish a connection with a database. Applications look up the data source on the Java Naming and Directory Interface (JNDI) tree or in the local application context and request a database connection to retrieve data. You must configure data sources for {KIE_SERVER} to ensure proper data exchange between the servers and the designated database.
+
+.Prerequisite
+{PRODUCT} for {JWS} is installed.
+
+.Procedure
+. Copy the following libraries to the `JWS_HOME/tomcat/lib` folder:
++
+[source]
+----
+org.jboss.spec.javax.transaction:jboss-transaction-api_1.2_spec
+org.jboss.integration:narayana-tomcat
+org.jboss.narayana.jta:narayana-jta
+org.jboss:jboss-transaction-spi
+org.jboss.logging:jboss-logging
+----
+. Copy your database JDBC driver to the `JWS_HOME/tomcat/lib` folder.
+. Configure the pooling XA data source:
++
+[NOTE]
+====
+Some of the properties in the following examples might not apply to your database server. Check the documentation for your JDBC driver to determine which properties to set.
+====
+.. Configure an XA data source without pooling capabilities. This XA data source is used to create new connections to the target database. In the following example, the XA datasourse is `xads` and the variables are defined in <<xa_data_source_{context}>>:
```

Comment:
Can we fix the passive voice here? As in, the SOMETHING uses the XA data source to...? (The "are defined" passive voice could be changes to "context defines" but not at all sure about that)

Reply:
I see your point. Leaving for now because I don't think it's terrible and need to wrap this up.